### PR TITLE
ClusterLoader - Add dynamic clients for measurement manager

### PR DESCRIPTION
**Pull request by Betula-L**
**Why we need it**:
https://github.com/kubernetes/perf-tests/issues/414

**What this PR does**:
Related to https://github.com/kubernetes/perf-tests/pull/423

### CODE REVIEW

File: clusterloader2/pkg/measurement/interface.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ import (
 type MeasurementConfig struct {
 	// Clientsets are kubernetes clients.
 	ClientSets *framework.MultiClientSet
+	// DynamicClients are kuberentes dynamic clients.
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/interface.go]: s/kuberentes/kubernetes/

Comment (Betula-L) [reply clusterloader2/pkg/measurement/interface.go]: Done. @wojtek-t

**Pull request merged**